docs/source/eFeatures.rst

```diff
@@ -2085,6 +2088,126 @@ with impedance_max_freq being a setting with 50.0 as a default value.
     else:
         return None
 
+activation_time_constant
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+`Python efeature`_ : Time constant for an ion channel activation trace.
+Fits for stim_start to trace maximum interval as A - B * exp(-t/tau).
+
+Attention! For voltage clamp data, user should pass the current response as voltage to efel.
+See voltage clamp example for more details.
+
+- **Required features**: time, voltage, stim_start, stim_end
+- **Units**: ms
+- **Pseudocode**: ::
+
+    def exp_fit(t, tau, A0, A1) -> np.ndarray | float:
+        return A0 + A1 * np.exp(-t / tau)
+
+    # isolate stimulus interval
+    stim_start_idx = np.flatnonzero(time >= stim_start)[0]
+    stim_end_idx = np.flatnonzero(time >= stim_end)[0]
+    time_interval = time[stim_start_idx:stim_end_idx]
+    voltage_interval = voltage[stim_start_idx:stim_end_idx]
+
+    # keep trace going from stim_start to voltage max
+    max_idx = np.argmax(voltage_interval)
+    time_interval = time_interval[:max_idx + 1]
+    voltage_interval = voltage_interval[:max_idx + 1]
+
+    # correct time so that it starts from 0
+    time_interval -= time_interval[0]
+
+    # fit
+    popt, _ = curve_fit(
+        exp_fit,
+        time_interval,
+        voltage_interval,
+        p0=(1., voltage_interval[-1], voltage_interval[0] - voltage_interval[-1]),
+        bounds=(((0, -np.inf, -np.inf), (np.inf, np.inf, 0))),  # positive tau, negative A1
```

Comment:
is the extra parenthesis inside `bounds` necessary?

Reply:
Fixed in latest commit